crates/kitsune-webfinger/tests/redirects.rs

```diff
@@ -64,12 +65,12 @@ async fn reject_fake_jrd_redirect() {
                     ..simd_json::from_slice(&mut base).unwrap()
                 })
                 .unwrap();
-                Ok::<_, Infallible>(Response::new(Body::from(body)))
+                Ok::<_, Infallible>(Response::new(BoxBody::new(Full::new(Bytes::from(body)))))
             }
             ("whitehouse.gov", "/.well-known/webfinger?resource=acct:0x0@whitehouse.gov") => {
                 Ok(Response::builder()
                     .status(StatusCode::NOT_FOUND)
-                    .body(Body::empty())
+                    .body(BoxBody::default())
```

Comment:
You can use the `Default` impl of `Full` instead, which creates an empty body.

Reply:
Applied